### PR TITLE
qcli 0.8 (new formula)

### PR DESCRIPTION
**Pull request by dericed**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/qcli.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+  depends_on "pkg-config" => :build
+  depends_on "qwt"
+  depends_on "qt"
+  depends_on "ffmpeg"
+
+  def install
+    ENV["QCTOOLS_USE_BREW"]="true"
+    path = ENV["PATH"]
+    ENV["PATH"] = "#{path}:#{HOMEBREW_PREFIX}/bin"
```
Comment (bfontaine) [inline Formula/qcli.rb]: Why do you need to modify the `PATH`?

File: Formula/qcli.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    cd "Project/QtCreator" do
+      cd "qctools-lib"
+      system "qmake", "qctools-lib.pro"
+      system "make"
+      cd "../qctools-cli"
```
Comment (bfontaine) [inline Formula/qcli.rb]: Please use `cd do` blocks instead of going back and forth.

File: Formula/qcli.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  desc "report audiovisual metrics via libavfilter"
+  homepage "https://bavc.org/preserve-media/preservation-tools"
+  url "https://github.com/bavc/qctools/archive/v0.8.tar.gz"
+  sha256 "5362dc8325aeb37e0742a5e5df7b831e7fe82a7b06c72c50463a43a7ad0b56bc"
+  head "https://github.com/bavc/qctools.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "qwt"
+  depends_on "qt"
+  depends_on "ffmpeg"
```
Comment (bfontaine) [inline Formula/qcli.rb]: Is `ffmpeg` needed outside of the test?

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/qcli.rb (hunk 3 above)
Comment (dericed) [inline Formula/qcli.rb]: Yes, it builds off ffmpeg.

File: Formula/qcli.rb (hunk 2 above)
Comment (dericed) [inline Formula/qcli.rb]: I simplified the cd'ing, rebased to squash, and push --force.

File: Formula/qcli.rb (hunk 1 above)
Comment (dericed) [inline Formula/qcli.rb]: Sorry, this may have been left in from testing. I removed the 2 path setting lines and it still seems to build okay.

**Comment by dericed** [comment]
Thanks. I've updated the patch according to @bfontaine's comments. It's ready for another look.

File: Formula/qcli.rb (hunk 3 above)
Comment (dunn) [inline Formula/qcli.rb]: If it's not needed at runtime, you can do `depends_on "ffmpeg" => :build`.

File: Formula/qcli.rb [hunk 4]
```diff
@@ -0,0 +1,36 @@
+class Qcli < Formula
+  desc "report audiovisual metrics via libavfilter"
```
Comment (dunn) [inline Formula/qcli.rb]: Could you capitalize `Report` please?

**Review by bfontaine — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @dericed for your contribution to Homebrew!

**Pull request closed without merging**